src/utils.js

```diff
@@ -35,3 +37,70 @@ export function buildDefinition( modelAttributeKey, options ) {
 
 	return definition;
 }
+
+export const FONT_COLOR = 'fontColor';
+export const FONT_BACKGROUND_COLOR = 'fontBackgroundColor';
+
+export function renderUpcastAttribute( styleAttr ) {
+	return viewElement => {
+		const fontColor = viewElement.getStyle( styleAttr );
+		return normalizeColorCode( fontColor );
+	};
+}
+
+export function renderDowncastElement( styleAttr ) {
+	return ( modelAttributeValue, viewWriter ) => viewWriter.createAttributeElement( 'span', {
+		style: `${ styleAttr }:${ modelAttributeValue }`
+	} );
+}
+
+function normalizeColorCode( value ) {
+	return value.replace( /\s/g, '' );
+}
+
+export function normalizeOptions( colorRow ) {
+	return colorRow
+		.map( getColorsDefinition )
+		.filter( option => !!option );
+}
+
+function getColorsDefinition( color ) {
```

Comment:
`getColorDefinition` if anything. But this has something to do with the conversion — maybe there's a better name for it?